server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -2994,7 +2994,12 @@ public String startVerification() {
      */
     private BlobContainer testContainer(String seed) {
         BlobPath testBlobPath;
-        if (prefixModeVerification == true) {
+
+        if (prefixModeVerification == true
+            && (clusterService.isStateInitialised() == false
```

Comment:
can this check pass for an older version node if its restarting in mixed opensearch version setup?

Reply:
No, it wont pass due to the added check now. Earlier it could pass this check even if all the nodes were not on the minimum supported version for prefix mode verification.